src/app/financial-chart/high-frequency/financial-chart-high-frequency.component.ts

```diff
@@ -0,0 +1,177 @@
+import {
```

Comment:
Having a 10ms update sample for finchart doesn't make any sense. Remove this until this is rethought.

Reply:
I don't thing having that frequent updates and an for counter makes sense for finchart where it doesn't have that kind of high frequency data, at least under usual conditions. Sample needs more thought.